### ISSUE TITLE
Log::Dispatch::Screen mangles Unicode text

### ISSUE THREAD

**Issue by mauke** [houseabsolute/Log-Dispatch#67]
https://metacpan.org/release/DROLSKY/Log-Dispatch-2.70/view/lib/Log/Dispatch/Screen.pm says:

> - utf8 (0 or 1)
>
> If this is true, then the output uses `binmode` to apply the `:encoding(UTF-8)` layer to the relevant handle for output.

Thus, if some other part of my code (or another module) already sets `binmode $handle, ":encoding(UTF-8)"`, this should be a harmless no-op. But:

```perl
use strict;
use warnings;
use Log::Dispatch;

binmode STDOUT, ":encoding(UTF-8)";

my $log = Log::Dispatch->new(
    outputs => [
        [
            'Screen',
            min_level => 'debug',
            newline   => 1,
            stderr    => 0,
            utf8      => 1,
        ],
    ],
);

$log->warning("caf\N{U+00E9}");
```

Actual output:
```
$ perl try.pl
cafÃ©
```

Expected output:
```
$ perl try.pl
café
```

**Comment by autarch** [comment]
Since it's actually using `STDOUT` or `STDERR` in the hood, contrary to the docs, this is the expected outcome. I'll update the docs to note this potential double-encoding issue.

**Comment by autarch** [comment]
Thanks for the bug report! The docs have been fixed in v2.71.
